newsService/src/test/java/com/kovan/NewsServiceTest.java

```diff
@@ -215,7 +215,7 @@ void testUpdateNewsInDb_ShouldUpdateExistingNews() throws Exception {
         when(restTemplate.getForObject(anyString(), eq(String.class))).thenReturn(apiResponse);
         when(objectMapper.readValue(anyString(), eq(NewsDto.class))).thenReturn(apiNewsDto);
         when(newsRepositoryService.saveNewsInDb(any(NewsDto.class))).thenReturn(existingNewsDto);
-        when(newsRepositoryService.findNewsInDb(eq(publishedAtDate))).thenReturn(of(existingNewsDto));
+        when(newsRepositoryService.findNewsInDb((publishedAtDate))).thenReturn(of(existingNewsDto));
```

Comment:
Here as well attach the screen shot

Reply:
Screenshot attached.

Reply:
![image](https://github.com/user-attachments/assets/f85b944a-949f-4f66-b946-2efcaabeb9b8)

Reply:
![image](https://github.com/user-attachments/assets/469540d9-8581-4b49-be8b-5d9539c5dde1)